### DISCUSSION TITLE
Testing campaign for TriliumNext beta

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Trilium#6061] [Releases]
@TriliumNext/everyone ,

The second beta of TriliumNext has been made available since yesterday: https://github.com/TriliumNext/Notes/releases/tag/v0.90.1-beta

We would appreciate any feedback regarding this release.

There are also big plans to convert the server to ES modules, so that will introduce another layer of regressions. If we catch things earlier, it will make the process easier.

Generally, the desktop application seems pretty stable and ready for day-to-day use.
You might have a small issue with the sync (hash failed during sync if you have removed items) but that will be solved in the next beta.

I also want to take the time to offer thanks to @deajan and @Nriver who tested v0.90.0-beta and found bugs.

**Comment by maphew** [comment]
nice to see the build-and-release infrastructure is working! 

Are there any considerations when testing the beta? For instance, is it okay to run on same machine alongside an older Trilium? Do they use the same database? What care should be taken to safeguard one's content? etc.

**Reply by eliandoran** [reply]
A manual backup should be OK. Generally they should be backwards compatible so it should be possible to work with both apps on the same database without issue.

On my side I have replaced Trilium Notes with Trilium Next at work fully, with no intention of coming back to the original.

**Comment by doobes** [comment]
Hey there,

Downloaded the beta.  I've tried all three .sh files and the Trilium file itself on a Zorin box from the command line.

I'm consistently getting:

Inconsistency detected by ld.so: ../sysdeps/x86_64/dl-machine.h: 534: elf_machine_rela_relative: Assertion `ELFW(R_TYPE) (reloc->r_info) == R_X86_64_RELATIVE' failed!

Cheers

chris

**Reply by meichthys** [reply]
Hi Chris, If you downloaded the beta from the link above, you're probably working on an older version. I suggest you try the latest release from here: https://github.com/TriliumNext/Notes/releases

If you want to test the bleeding edge, you can pick the latest release from here: https://github.com/TriliumNext/Notes/actions?query=branch%3Adevelop+event%3Apush

**Reply by perfectra1n** [reply]
After doing some quick googling with that error, it looks like your machine might need a reinstall if trying newer versions doesn’t help…

https://www.reddit.com/r/pop_os/comments/r0ala2/inconsistency_detected_by_ldso/

**Comment by doobes** [comment]
Welp,

Tried v0.90.4.  Same thing.

It's a standard Zorin install on a Dell XPS 13 laptop. Dunno anything about SELinux. Sorry....

I'm just extracting to a folder, and then chmod-ing the *.sh files and executing.

cheers

**Reply by perfectra1n** [reply]
It looks like you might need to reinstall your Zorin and/or upgrade your kernel:

https://talk.plesk.com/threads/inconsistency-detected-by-ld-so-during-apache-reload-restart.362277/

https://www.reddit.com/r/pop_os/comments/r0ala2/inconsistency_detected_by_ldso/

https://gitlab.freedesktop.org/drm/i915/kernel/-/issues/2970

**Comment by doobes** [comment]
Dunno about any of that. I'm **_DEFINITELY_** not reinstalling Zorin....

Running it on my ElementaryOS box,

using ./trilium-portable.sh"
```
(node:78982) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `trilium --trace-warnings ...` to show where the warning was created)
A JavaScript error occurred in the main process
Uncaught Exception:
SqliteError: database is locked
    at Database.pragma (/mnt/library/Linux/Trilium/trilium-linux-x64-0.90.4/resources/app.asar/node_modules/better-sqlite3/lib/methods/pragma.js:11:44)
    at file:///mnt/library/Linux/Trilium/trilium-linux-x64-0.90.4/resources/app.asar/src/services/sql.js:23:18
(node:78982) UnhandledPromiseRejectionWarning: SqliteError: database is locked
    at Database.pragma (/mnt/library/Linux/Trilium/trilium-linux-x64-0.90.4/resources/app.asar/node_modules/better-sqlite3/lib/methods/pragma.js:11:44)
    at file:///mnt/library/Linux/Trilium/trilium-linux-x64-0.90.4/resources/app.asar/src/services/sql.js:23:18
(node:78982) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
MESA-INTEL: warning: Haswell Vulkan support is incomplete
[79016:0813/114248.771262:ERROR:vaapi_wrapper.cc(1238)] Empty codec maximum resolution
[79016:0813/114248.771453:ERROR:vaapi_wrapper.cc(1147)] FillProfileInfo_Locked failed for va_profile VAProfileH264ConstrainedBaseline and entrypoint VAEntrypointVLD
[79016:0813/114248.771549:ERROR:vaapi_wrapper.cc(1238)] Empty codec maximum resolution
[79016:0813/114248.771616:ERROR:vaapi_wrapper.cc(1147)] FillProfileInfo_Locked failed for va_profile VAProfileH264Main and entrypoint VAEntrypointVLD
[79016:0813/114248.771681:ERROR:vaapi_wrapper.cc(1238)] Empty codec maximum resolution
[79016:0813/114248.771746:ERROR:vaapi_wrapper.cc(1147)] FillProfileInfo_Locked failed for va_profile VAProfileH264High and entrypoint VAEntrypointVLD
```
Using ./trilium-no-cert-check.sh and ./trilium-safe-mode.sh work fine.

I'll play with it and report back

Thanks for all the work guys, you rock!

chris

**Comment by doobes** [comment]
I exported most of my Trilium data from the previous version and imported into the beta.

Everything looks fine.  I moved some stuff around and futzed with the data.  All good.

One thing I noticed (about the old version) is when you export something, the file window pops up behind the Trilium window on this Linux box.  It does not behave this way on my Windows install.

I'll keep playing with the beta and report back.
